src/pages/authorization/index.mdx

```diff
@@ -44,6 +44,15 @@ If you'd like to see the authorization flows explained live, watch our video tut
   allowfullscreen
 ></iframe>
 
+# Sample Authorization Apps
+
+We've prepared two apps with fully implemented authorization flows. You can test them, experiment with them, or modify their code and extend their functionality.
+
+The apps demonstrate how to implement two flows from the [LiveChat Accounts SDK](/authorization/sign-in-with-livechat#accounts-sdk): the [popup flow](/authorization/sign-in-with-livechat#popup-1) and the [redirect flow](/authorization/sign-in-with-livechat#redirect-1).
+
+Please check out our Sample authorization app tutorial if you want more information about our sample apps.
```

Comment:
Do we already know what the link here will be?

Reply:
https://developers.livechat.com/updates/learn-authorization-with-sample-apps